R/MeasureFairness.R

```diff
@@ -0,0 +1,66 @@
+#' @title Fairness Measure
+#' @name mlr_measures_fairness
+#'
+#' @description
+#' This measure specializes [Measure] to allow for measuring statistical group fairness:
```

Comment:
Please check that the link to measure works. I would assume it needs an `(mlr3)`?

Reply:
Fixed the hyperlink